projects/igniteui-angular/src/lib/progressbar/progressbar.component.ts

```diff
@@ -455,6 +456,24 @@ export class IgxCircularProgressBarComponent extends BaseProgress {
     @Input()
     public text: string;
 
+    /**
+     * Sets the text template to be displayed inside the `igxCircularBar`..
+     * ```html
+     * <ng-template #textTemplate>
+     *     <svg:tspan class="progress-circular__text">20</tspan>
+     *     <svg:tspan >Precess</tspan>
+     * </ng-template>
+     * ```
+     * ```typescript
+     * @ViewChild("'textTemplate'", {read: TemplateRef })
+     * public textTemplate: TemplateRef<any>;
+     * this.circularBar.textTemplate = this.textTemplate;
+     * ```
+     * @memberof IgxCircularProgressBarComponent
+     */
+    @Input()
+    public textTemplate: TemplateRef<any>;
```

Comment:
The `@Input` is great, however it would be even better to be able to instantiate the template directly inside the `igx-circular-bar` tag. For example:
```html
<igx-circular-bar>
    <ng-template let-value>…</ng-template>
</igx-circular-bar>
```